### ISSUE TITLE
osc bundle for /channel/(x)/stage/layer/(y)/file/frame should be expanded

### ISSUE THREAD

**Issue by tuxilus** [CasparCG/Server#446]
Hi
for partially played out files (seek=3000, frames =6000)
it is impossible to parse out the remaining time for such files from the osc stream because you dont get startframes and number of frames. I am actually fetching this information via a telnetconection to the server. info 3-20 for example.
By parsing the xml i get 
nb_frames
frames-left
file-nb-frames

Some arithmetic and I have all I need to cue a video out for the speaker. The transition is done automated.

"Attention please! Exit in ten seconds...
ten, nine, eight" ... and so on.

I suggest to send the following osc bundle to registered clients.
/channel/3/stage/layer/20/file/frame
{
actual frame int,
total frames int,
number of playbackframes int, (new)
playback frames left (new)
}

I for myself am only parsing channel 3, layer 20 because this is my playout channel for videos.
Now I, am sending an osc message to the countdown Megaclock to avoid permanent parsing via telnet.

If I am playing out three short cuts from 1 long video for example songs then it is impossible to initiate an automated lookup. therefore this extension is really of great use for the osc message system, (does mean also for the ccaspar client to show sensefull informations.

Thanks for reading
Horst verbic

**Comment by ronag** [comment]
@5opr4ni this is what we discussed yesterday.

I suggest:

/file/time
/time
/file/frame
/frame

**Comment by 5opr4ni** [comment]
@ronag Yes! We also talked about this today. I agree to your suggestion. It will be impossible for a solution listening and monitoring the server to show correct data, if that solution hasn't issued the command.
So this would solve that problem.

**Comment by TomKaltz** [comment]
I highly recommend adding something like an correlation ID to producers like I discussed in...https://github.com/CasparCG/server/issues/850#issuecomment-367071134

Also I realized there is no way to tell if a producer is loaded on the background of a layer.  OSC definitely needs some work for 2.2.
